mappers/bluetooth_mapper/watcher/watcher.go

```diff
@@ -0,0 +1,176 @@
+/*
+Copyright 2019 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package watcher
+
+import (
+	"errors"
+	"strings"
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/paypal/gatt"
+
+	"github.com/kubeedge/kubeedge/mappers/bluetooth_mapper/action_manager"
+	"github.com/kubeedge/kubeedge/mappers/bluetooth_mapper/data_converter"
+	"github.com/kubeedge/kubeedge/mappers/bluetooth_mapper/helper"
+)
+
+var DeviceConnected = make(chan bool)
+var done = make(chan struct{})
+var deviceName string
+var deviceID string
+var actionManager []action_manager.Action
```

Comment:
see previous comment for package naming. same for data_converter

Reply:
done, thanks